repo/incubating/flink/versions/0/flink-frameworkversion.yaml

```diff
@@ -81,9 +194,29 @@ spec:
           spec:
             containers:
             - name: jobmanager
-              image: flink:1.7
+              image: kudobuilder/flink:1.7.2
+              command: ["/opt/flink/bin/flink-console.sh"]
```

Comment:
I had to do this as a workaround for the current docker-entrypoint.sh.  The script doesn't allow the passing of additional configurations outside of webport (-r) and configDir.